workflow-report/README.md

```diff
@@ -0,0 +1,44 @@
+# Github Action: report a failed workflow to VK Team chat
+This action sends the following information to VK Team chat:
+* __Job__: name of the failed job with a link to this job
+* __Commit__: hash of the commit started the job with a link to this commit
+* __Branch__: name of the branch
+* __History__: link to commit history
+* __Triggered on__: name of the event triggered the workflow
+* __Committer__: GitHub login of commit author
+* __Commit message subject__: subject (first string) of the commit message
+* __Steps info__: failed steps id and info about the steps (outputs, outcome, conclusion)
```

Comment:
1. `__Failed steps info__`
2. `: info about the failed steps (outputs, outcome, conclusion)`